packages/core/test/ArrayField.test.jsx

```diff
@@ -1294,14 +1337,16 @@ describe('ArrayField', () => {
       it('should handle a change event', () => {
         const { node, onChange } = createFormComponent({ schema });
 
-        Simulate.change(node.querySelector('.field select'), {
-          target: {
-            options: [
-              { selected: true, value: 0 }, // use index
-              { selected: true, value: 1 }, // use index
-              { selected: false, value: 2 }, // use index
-            ],
-          },
+        act(() => {
+          Simulate.change(node.querySelector('.field select'), {
```

Comment:
Can you make this use `fireEvent` instead?
```suggestion
          fireEvent.change(node.querySelector('.field select'), {
```